Compile/interface/ZJetSettings.h

```diff
@@ -93,6 +93,9 @@ class ZJetSettings : public KappaSettings
     IMPL_SETTING(float, CutGenHTMax)
     IMPL_SETTING(float, GenZMassRange)
     IMPL_SETTING(std::string, CutEtaPhiCleaningFile)
+    IMPL_SETTING_DEFAULT(bool, CutVetoCleanedEvents, false)  // true: cleaned events are vetoed according to the next two settings
+    IMPL_SETTING_DEFAULT(float, CutVetoJetsAbove, -1.0)  // default: all        
+    IMPL_SETTING_DEFAULT(int, CutVetoNJets, 1)  // default: only check leading jet; to check all jets, use 999
```

Comment:
This can in principle be bigger than 999, since it is an `int`. Maybe remove this comment.